README.md

```diff
@@ -49,38 +47,41 @@ unless real values don't help users know what to change.
 -->
 
 ```hcl
-
+module "service_auth_cbr_rules" {
+  # Replace "main" with a GIT release version to lock into a specific release
+  source                = "git::https://github.com/terraform-ibm-modules/terraform-ibm-s2s-auth.git?ref=main"
```

Comment:
can you use hashicorp syntax here, as we will push to hashicorp once first release is out